tutorial/10_key_features/005_visualization.py

```diff
@@ -77,7 +77,7 @@ def objective(trial):
     # Add a callback for pruning.
     pruning_callback = optuna.integration.LightGBMPruningCallback(trial, "auc")
     gbm = lgb.train(
-        param, dtrain, valid_sets=[dvalid], verbose_eval=False, callbacks=[pruning_callback]
+        param, dtrain, valid_sets=[dvalid], callbacks=[pruning_callback, lgb.log_evaluation(0)]
```

Comment:
@nzw0301 Could you explain what will happen if you pass 0 to `period` argument of `log_evaluation` function? I couldn't find that in [the documentation](https://lightgbm.readthedocs.io/en/latest/pythonapi/lightgbm.log_evaluation.html)🙇‍♂️

Reply:
if `period` is non-positive, `log_evaluation` disable the logging because the first condition is false in 
https://github.com/microsoft/LightGBM/blob/5df7d516f3e64e8431aecfea892a850afeb787bb/python-package/lightgbm/callback.py#L75

Reply:
Hmm, if we do not want logging, why do we register this callback?

Reply:
Ah, good point. I'll update the PR by removing it. Should I apply a similar change to the test code too? We might be able to remove `log_evaluation(-1)` in https://github.com/optuna/optuna/pull/3086/files.

Reply:
👍 
Thank you!

Reply:
I tried LightGBMTuner without `log_evaluation`, but it didn't work like this example. So I don't update the this PR to remove `log_evaluation` for LightGBMTuner tests.